src/occwl/compound.py

```diff
@@ -1,32 +1,127 @@
-from OCC.Core.TopoDS import (
-    topods,
-    TopoDS_Wire,
-    TopoDS_Vertex,
-    TopoDS_Edge,
-    TopoDS_Face,
-    TopoDS_Shell,
-    TopoDS_Solid,
-    TopoDS_Shape,
-    TopoDS_Compound,
-    TopoDS_CompSolid,
-    topods_Edge,
-    topods_Vertex,
-    TopoDS_Iterator,
-)
-
+from OCC.Core.TopoDS import TopoDS_Compound, TopoDS_CompSolid
+from OCC.Extend.DataExchange import read_step_file, list_of_shapes_to_compound
+from occwl.shape import Shape
 from occwl.solid import Solid
+from occwl.face import Face
+from occwl.wire import Wire
+from occwl.edge import Edge
+from occwl.vertex import Vertex
 
 
-class Compound(Solid):
+class Compound(Shape):
```

Comment:
Shouldn't we have a base class for the methods which `Solid` and `Compound` want to share.  For example it's useful to be able to get triangles from all the solids in a compound without having to write special code.